### ISSUE TITLE
Factoid 

### ISSUE THREAD

**Issue by qjhart** [ucd-library/ucdlib-theme#22]
Kmmy Identified the following factoid item that could be included generally.

It is the `factoid` and can be seen at: [UC Davis About](https://www.ucdavis.edu/about) and also in [campus Life](https://www.ucdavis.edu/campus-life).  The component consists of an icon, title and statistic.  The build out of the component on those sites, uses specialized div tags for the brackets.

The link,statistic and title might need to be dynamically updated, but that is the limit of the javascript use.

```html
<div class="factoid">
		<a href="https://rec.ucdavis.edu/">
			<div class="factoid-bracket left">
				<div class="factoid-bracket__thing"></div>
			</div>
			<div class="factoid-content">
								<div class="factoid-content__icon">
					<span class="fas fa-dumbbell fa-3x"></span>
				</div>
								<div class="factoid-content__text">
					<p class="factoid-content__text--statistic">150,000</p>
					<p class="factoid-content__text--title">square foot fitness center</p>
				</div>
			</div>
			<div class="factoid-bracket right">
				<div class="factoid-bracket__thing"></div>
			</div>
		</a>
	</div>
```

**Comment by qjhart** [comment]
@spelkey-ucd will update this issue for the desired SASS files, and then @wrenaria will ask Jessica for help on that.

**Comment by spelkey-ucd** [comment]
Yes, if there is a public repository with either sass or css partials we could leverage, that would go a long way to ensuring we stay on brand.

Something like this: https://bitbucket.org/ietwebdev/sitefarm-pattern-lab-one/src/redesign/source/sass/

**Comment by spelkey-ucd** [comment]
@sbagg I have uploaded the css to the [theme-sass repo](https://github.com/ucd-library/ucdlib-theme-sass/tree/main/source/sass/main_site/factoid).

I also included the twig file, just for reference

**Comment by qjhart** [comment]
@sbagg there remain a few small tweeks to the [documentation page](https://github.com/ucd-library/ucdlib-theme/blob/main/test-app/pages/page-infographics.tpl.js)  

- [ ] The layout for the columns of factoids should use standard ucd layout themeing and not a one-off grid.
- [ ] We should  add an bit of example javascript that dynamically updates an incon.  For example adding in a statistic for time, or something, and showing it's updates.
